compiler/AST/LoopExpr.cpp

```diff
@@ -415,6 +411,15 @@ static FnSymbol* buildSerialIteratorFn(const char* iteratorName,
   return sifn;
 }
 
+static Expr* buildLIFNwhere(ArgSymbol* lifnTag, ArgSymbol* lifnIterator,
```

Comment:
What's a LIFN ? I'd go for a longer name, but you definitely need to add a comment.